js/header_dialog.js

```diff
@@ -964,6 +1030,22 @@ function HeaderDialog(dialog, onSave) {
                         }
                 });
 
+                // Scan the pid_bandwidth table
+                $(".pid_bandwidth input").each(function() {
+                        if($(this).val()!=null) {
+                                if($(this).attr('decPl')!=null) {
+                                        var matches=$(this).attr('name').match(/(.+)\[(\d+)\]/);
```

Comment:
Be very careful using 'var'.
This should be 'let'

Reply:
This function is not required and does not get used. Removing.